packages/react-native-tab-view/src/PagerViewAdapter.tsx

```diff
@@ -3,6 +3,7 @@ import { Animated, Keyboard, Platform, StyleSheet } from 'react-native';
 import ViewPager, {
   type PageScrollStateChangedNativeEvent,
 } from 'react-native-pager-view';
+import type { NativeProps } from 'react-native-pager-view/lib/typescript/specs/PagerViewNativeComponent';
```

Comment:
Don't import private files, e.g. `react-native-pager-view/lib/typescript`. You can get the type with `React.ComponentProps<typeof ViewPager>['onPageSelected']`